tests/Sharpliner.Tests/GitHub/WorkflowTests.cs

```diff
@@ -291,5 +291,48 @@ public void Workflow_Detailed_Permissions()
             Assert.Contains(GitHubPermissionScope.Actions, w.Permissions.Read);
             Assert.Contains(GitHubPermissionScope.Contents, w.Permissions.Read);
         }
+
+        [Fact]
+        public void Workflow_No_Environment_Variables()
+        {
+            var w = new Workflow
+            {
+                On =
+                {
+                    Webhooks =
+                    {
+                        new PullRequest
+                        {
+                            Activities = { PullRequest.Activity.Assigned, PullRequest.Activity.Closed }
+                        }
+                    }
+                },
+            };
+            Assert.Empty(w.Enviroment.Keys);
+        }
+
+        [Fact]
+        public void Workflow_Environment_Variables()
+        {
+            var w = new Workflow
+            {
+                On =
+                {
+                    Webhooks =
+                    {
+                        new PullRequest
+                        {
+                            Activities = { PullRequest.Activity.Assigned, PullRequest.Activity.Closed }
+                        }
+                    }
+                },
+                Enviroment =
+                {
+                    ["Database"] = "production",
```

Comment:
Ah cool, I will change my env definitions to this style.

We should then also give some styleguide so that people also write clean definitions

Reply:
I saw yours, I liked it. But then I make lazy lol